src/main/java/com/brew/oauth20/server/controller/AuthorizeController.java

```diff
@@ -80,7 +80,7 @@ private ResponseEntity<String> authorize(AuthorizeRequestModel authorizeRequest,
                     .getService(ResponseType.fromValue(authorizeRequest.getResponse_type()));
 
             var authorizeTypeValidationResult = authorizeTypeProvider.validate(authorizeRequest.getClient_id(),
-                    authorizeRequest.getRedirect_uri());
+                    authorizeRequest.getRedirect_uri(), null);
```

Comment:
[874 875 Diff](https://github.com/BrewInteractive/oauth-server/compare/TMID-875-refactor-validate-method-in-client-validator-in-o-auth-server-project...TMID-874-refactor-validate-method-in-base-authorize-type-provider-in-o-auth-server-project#diff-32ec35518a0a6b8e09acfecd13afc0128392b9e3fd125b26e6dfa0847b2c0a5eR83)
null parameter is still not clear on my side. it still exists with both pr (875, 874)

Reply:
will be refactored in next tasks